Bottom: baedc7b09701a5ff113e7e74a5c3eafeb4a23d68
Top:    5e4d5b81c4066ed0dba3f1d37ddd49b9ac809822
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-13 14:55:58 +0800

Refresh of virtio-net-map-offload

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 2d98105..8cde2bc 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -2596,7 +2596,7 @@ static int virtnet_bfp_ctrl_entry_op(struct bpf_offloaded_map *offmap,
 				     u8 *out_key, u8 *out_value)
 {
 	struct bpf_map *map = &offmap->map;
-	struct virtio_net_ctrl_ebpf_map *ctl = &vq->ctrl->ebpf;
+	struct virtio_net_ctrl_ebpf_map *ctrl = &vi->ctrl->ebpf;
 	struct scatterlist sg;
 
 	/* Copy inputs */
